src/commands/force/org/shape/create.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { EOL } from 'os';
+import { SfdxCommand } from '@salesforce/command';
+import { SfdxError, Messages, Connection } from '@salesforce/core';
+import { RecordResult } from 'jsforce';
+import { isShapeEnabled, JsForceError } from '../../../../shared/orgShapeListUtils';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-signups', 'shape.create');
+
+const commandTimeOutInMS = 30e3;
+
+export interface ShapeCreateResult {
+  shapeId: string;
+  shapeFile: string;
+  success: boolean;
+  errors: [];
+}
+
+export class OrgShapeCreateCommand extends SfdxCommand {
+  public static readonly description = messages.getMessage('create_shape_command_description');
+  public static readonly examples = messages.getMessage('create_shape_command_help').split(EOL);
+  public static readonly requiresUsername = true;
+  public static readonly varargs = true;
```

Comment:
This command doesn't use `varargs`, but if we remove now it could break users passing a vararg.